misc/version_updater.py

```diff
@@ -462,23 +462,23 @@ def check_tool(tool: Tool, update: bool) -> VersionUpdateResult:
     version = TOOLS_VERSION[tool]
     update_res = VersionUpdateResult([], set())
 
-    for filename, tools_in_file in FILES_WITH_VERSION_INFO.items():
+    for glob_path, tools_in_file in FILES_WITH_VERSION_INFO.items():
         regexes = tools_in_file.get(tool, None)
         if regexes is None:
             continue
 
-        matched_files = glob.glob(str(filename), recursive=True)
-        if len(matched_files) > 0:
-            for file in matched_files:
-                file = Path(file)
-                if update:
-                    res = update_tool_version(file, regexes, version)
-                    update_res.errors += res.errors
-                    update_res.updated.update(res.updated)
-                else:
-                    update_res.errors += check_tool_version(file, regexes, version).errors
-        else:
-            update_res.errors.append(f"No match with regex `{filename}`")
+        matched_files = glob.glob(str(glob_path), recursive=True)
+        if len(matched_files) == 0:
```

Comment:
If glob returns an empty list in case of no match: 
```suggestion
        if not matched_files:
```